### PR TITLE
google-chrome: add page

### PR DESCRIPTION
**Pull request by navarroaxel**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/linux/google-chrome.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# google-chrome
+
+> The web browser from Google.
+> More information: <https://chrome.google.com>.
+
+- Run with a temp profile:
+
+`google-chrome --user-data-dir=/tmp/chrome_dev`
```
Comment (einverne) [inline pages/linux/google-chrome.md:8]:
```suggestion
`google-chrome --user-data-dir={{path/to/chrome_dev}}`
```

File: pages/linux/google-chrome.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+- Run without CORS validation, useful to test an API:
+
+`google-chrome --user-data-dir=/tmp/chrome_dev --disable-web-security`
```
Comment (einverne) [inline pages/linux/google-chrome.md:12]:
```suggestion
`google-chrome --user-data-dir={{path/to/chrome_dev}} --disable-web-security`
```

File: pages/linux/google-chrome.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
+# google-chrome
+
+> The web browser from Google.
+> More information: <https://chrome.google.com>.
+
+- Run with a temp profile:
```
Comment (einverne) [inline pages/linux/google-chrome.md:6]:
```suggestion
- Run with a temporary profile:
```
I've looked up the oxford dictionary, `temp` is not a valid word.

**Comment by navarroaxel** [comment]
@einverne I thought, like info, is a common accepted word, but you're right!

The use of `/tmp` is useful because is cleaned after a reboot in Linux.

Suggestions applied!

File: pages/linux/google-chrome.md [hunk 4]
```diff
@@ -0,0 +1,12 @@
+# google-chrome
+
+> The web browser from Google.
+> More information: <https://chrome.google.com>.
+
+- Run with a temporary profile:
```
Comment (zdroid) [inline pages/linux/google-chrome.md:6]: But it's not temporary, as you can run it again with the same command (unless `path/to/chrome_dev` is deleted).

File: pages/linux/google-chrome.md [hunk 5]
```diff
@@ -0,0 +1,12 @@
...
+
+`google-chrome --user-data-dir={{path/to/chrome_dev}}`
+
+- Run without CORS validation, useful to test an API:
+
+`google-chrome --user-data-dir={{path/to/chrome_dev}} --disable-web-security`
```
Comment (zdroid) [inline pages/linux/google-chrome.md:12]: Isn't it more clear like this?
```suggestion
`google-chrome --disable-web-security`
```

Comment (navarroaxel) [reply pages/linux/google-chrome.md:12]: Is required:
```bash
$ google-chrome --disable-web-security
[0801/110723.707530:ERROR:chrome_main_delegate.cc(663)] Web security may only be disabled if '--user-data-dir' is also specified with a non-default value.
```

File: pages/linux/google-chrome.md (hunk 4 above)
Comment (navarroaxel) [reply pages/linux/google-chrome.md:6]: Do you want I change the text to `Run with a custom profile directory`?

Comment (zdroid) [reply pages/linux/google-chrome.md:6]: Yep, that's good.

Comment (navarroaxel) [reply pages/linux/google-chrome.md]: Done!

**Comment by sbrl** [comment]
Merging, since everyone's comments have been addressed I think

**Comment by sbrl** [comment]
Next time, perhaps you could avoid force-pushing please? It makes it harder to tell what's changed when reviewing, and we've got a special button that allows us to squash easily on merge :smiley_cat:

**Pull request merged**